### PR TITLE
feat: added query param to info request to optionally add history (closes #8)

### LINKED ISSUE #8: Feature: to add the rating history to the full info call
I wanted to get the full player information & rating history in a single request ...

**Before** `/player/418250/info`
```
{
   name: "Webb, Matthew D ",
   federation: "England",
   birth_year: 1986,
   sex: "Male",
   title: "None",
   standard_elo: "2249",
   rapid_elo: "Notrated",
    blitz_elo: "Notrated",
    world_rank_all_players: 13860,
    world_rank_active_players: 0,
    national_rank_all_players: 237,
    national_rank_active_players: 0,
    continental_rank_all_players: 10563,
    continental_rank_active_players: 0
}
```
**After**  `/player/418250/info?include_history=true`
```
{
    name: "Webb, Matthew D ",
    federation: "England",
    birth_year: 1986,
    sex: "Male",
    title: "None",
    standard_elo: "2249",
    rapid_elo: "Notrated",
    blitz_elo: "Notrated",
    world_rank_all_players: 13860,
    world_rank_active_players: 0,
    national_rank_all_players: 237,
    national_rank_active_players: 0,
    continental_rank_all_players: 10563,
    continental_rank_active_players: 0,
    history: [ {
        date: "2020-Sep",
        numeric_date: 202009,
        standard: "2249",
        num_standard_games: "0",
        rapid: "",
        num_rapid_games: "",
        blitz: "",
        num_blitz_games: ""
   }, ... ]
}
```

### PR DESCRIPTION
**Pull request by matt-d-webb**
Closes #7 

Code change doesn't impact tests, perhaps it should 😕

image

Will update the `README` in this PR, if approved.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -39,13 +39,15 @@ const getPlayerElo = async (fide_num) => {
  * @param {Integer} fide_num
  * @returns {JSON} Player full information
  */
-const getPlayerFullInfo = async (fide_num) => {
+const getPlayerFullInfo = async (fide_num, include_history = false) => {
     const { data } = await fetchProfilePage(fide_num);
+    const history = await (include_history ? getPlayerHistory(fide_num) : Promise.resolve(undefined));
```
Comment (matt-d-webb) [inline src/main.js:44]: Definitely alternative ways for achieving this.

**Comment by xRuiAlves** [comment]
Hello @matt-d-webb :wave: 

Thank you for taking the time to suggest this change and implement it! It looks good to me, working as intended.

Regarding the tests, you are completely right. At the moment the test suite only covers the HTML parsing itself and some utility functions. I'll work on improving it in the future! :+1: 

If you could add the `include_history` usage information to the `README`, I'll be happy to approve this PR!

File: src/main.js (hunk 1 above)
Comment (xRuiAlves) [reply src/main.js:44]: True! But I'm fine with accepting this 1-liner :stuck_out_tongue:

**Review by xRuiAlves — APPROVED** [review]
LGTM :+1:

**Pull request merged**